### PR TITLE
Douglas/mavlink passthrough

### PR DESCRIPTION
**Pull request by douglaswsilva**
- Add support to mavlink passthrough
- Send mavlink commands to drone
- Send mavlink messages to drone

### CODE REVIEW

File: src/plugins/passthrough/passthrough_impl.cpp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    message.magic = mavlink_message.magic;
+    message.len = mavlink_message.len;
+    message.incompat_flags = mavlink_message.incompat_flags;
+    message.compat_flags = mavlink_message.compat_flags;
+    message.seq = mavlink_message.seq;
+    message.sysid = mavlink_message.sysid; 
+    message.compid = mavlink_message.compid;
+    message.msgid = mavlink_message.msgid;
+
+    for(int i = 0; i < 33; i++) {
```
Comment (byuarus) [inline src/plugins/passthrough/passthrough_impl.cpp:40]: Why 33?

**Review by byuarus — COMMENTED** [review]
I have a question

File: src/plugins/passthrough/passthrough_impl.cpp [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        message.payload64[i] = mavlink_message.payload64[i];
+    }
+
+    for(int i = 0; i < 2; i++) {
```
Comment (Anhtastic) [inline src/plugins/passthrough/passthrough_impl.cpp:44]: I just want to sound like I'm contributing, so I'm copying @byuarus , why 2? Jk lol.. I don't understand any of this.

**Review by Anhtastic — APPROVED** [review]
On a scale of 1 - 10 on how much I understood this, I'd give myself a 0.

File: src/plugins/passthrough/passthrough_impl.cpp (hunk 1 above)
Comment (douglaswsilva) [reply src/plugins/passthrough/passthrough_impl.cpp:40]: 33 is the fixed size of the array for payload.

**Pull request merged**